src/CommonDBTM.php

```diff
@@ -4722,6 +4722,7 @@ public static function getSpecificValueToDisplay($field, $values, array $options
             case '_virtual_datacenter_position':
                 $static = new static();
```

Comment:
Maybe this could do the job.
```suggestion
                /** @var CommonDBTM|\Glpi\Features\DCBreadcrumb $static */
                $static = new static();
```

Reply:
I tried already in facts... But since this is a trait; that's not working: `PHPDoc tag @var for variable $static has invalid type Glpi\Features\DCBreadcrumb.`